fiat-rust/Cargo.toml

```
@@ -1,8 +1,13 @@
 [package]
 name = "fiat-rust"
```

```suggestion
 name = "fiat-crypto"
```
I think fiat-rust is too general, either we should call it fiat-crypto or fiat-crypto-rust.  Thoughts?

---

Huh, `*-rust` goes against the [naming guidelines](https://rust-lang.github.io/api-guidelines/naming.html) so I guess that leaves us with `fiat-crypto`. I'll make the change.